docs/policies/github.md

```diff
@@ -5,8 +5,15 @@
 | Identity & Access Management | github | MEDIUM | Repository is Not Private. | accurics.github.IAM.1 |
 
 
+### github_repository_webhook
+| Category | Resource | Severity | Description | Reference ID |
+| -------- | -------- | -------- | ----------- | ------------ |
+| Encryption and Key Management | github | MEDIUM | Insecure SSL is used for repository webhook. | accurics.github.EKM.2 |
+
+
 ### github_organization_webhook
 | Category | Resource | Severity | Description | Reference ID |
 | -------- | -------- | -------- | ----------- | ------------ |
 | Encryption and Key Management | github | MEDIUM | Insecure SSL is used for organization webhook. | accurics.github.EKM.1 |
-| Encryption and Key Management | github | MEDIUM | Insecure SSL is used for repository webhook. | accurics.github.EKM.2 |
```

Comment:
The naming convention is different for github? accurics.github.EKM.2 vs AWS.LambdaFunction.Logging.0472 (referring to the "accurics" prefix).

Reply:
This was generated with the python script on [/docs](https://github.com/accurics/terrascan/blob/master/docs/policies/document_policies.py). We'll need to fix the policy IDs separately.